lib/parse/fuzz/fuzz_targets/parse.rs

```diff
@@ -0,0 +1,7 @@
+#![no_main]
```

Comment:
Is this fuzzing pen? It might be, I can't tell from the PR alone.

Reply:
yes, they broke up their compiler into different crates that fulfill different parts of the compilation chain. This is the parser (and comment parser, for doc comments I suppose) frontend